code/modules/supermatter/supermatter.dm

```diff
@@ -159,6 +187,10 @@ var/list/supermatter_final_thoughts = list(
 	var/aw_delam = FALSE
 	var/aw_EPR = FALSE
 
+	var/datum/sound_token/sound_token
+	var/sound_id
+	var/working_sound
```

Comment:
Not clear on why you track `working_sound` at all since it only seems to be passed to the callback as an arg.